.github/workflows/static-analysis.yml

```diff
@@ -90,7 +90,7 @@ jobs:
                   php redaxo/bin/console package:install structure/history --ansi
                   php redaxo/bin/console package:install structure/version --ansi
             - run: |
-                  vendor/bin/psalm --taint-analysis
+                  vendor/bin/psalm --use-baseline=.tools/psalm/baseline-taint.xml --taint-analysis --no-cache
```

Comment:
lokal funktionierts.. ich verstehe nicht warum die baseline nicht in den github actions funktioniert.. vllt ist es es WIN vs. LINUX problem.

@gharlan ggf. kannst du die taint-baseline mal auf deinem Mac generieren, ggf. gehts dann?

Reply:
hat sich erledigt, nachdem ich lokal psalm aktualisiert hab und die baseline auf dem aktuellen release generiert hab